rbc/tests/heavydb/test_heavydb_thrift.py

```diff
@@ -33,15 +33,15 @@ def foo(x, y):
 def test_get_udtf_details(heavydb):
     heavydb.unregister()
 
-    @heavydb('int32(Column<int>, OutputColumn<int>)')
+    @heavydb('int32(Column<int>, OutputColumn<int>)', devices=['cpu'])
```

Comment:
Can this UDTF run in the GPU?

Reply:
Yes, it can. This changeset was introduced to make the test stable: when both CPU and GPU would be enabled, then `len(details)` would be 2 and the assert statement below would fail.